### ISSUE TITLE
Remove docker builds for Day 5

### ISSUE THREAD

**Issue by cdennig** [azuredevcollege/trainingdays#142]
These are not needed for day 5 (AFAIK) and will make the checkpoint deployment much easier.

CC: @waeltken

**Comment by cdennig** [comment]
Let's discuss on the next meeting.

**Comment by waeltken** [comment]
Yes, they can be removed. Especially from the checkpoint they _should_ be removed.
